examples/simple.jsx

```diff
@@ -29,12 +29,12 @@ class TestChild extends React.Component { // eslint-disable-line react/no-multi-
 class Test extends React.Component { // eslint-disable-line react/no-multi-comp
   render() {
     return (
-      <BeaconConfig persistent>
+      <BeaconConfig>
```

Comment:
These were useful for testing persistence, not so useful now. This stuff should really be tested with E2E tests.